JDBC/projectTest/TestQueryClass.java

```diff
@@ -0,0 +1,70 @@
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import pOJOClasses.Title;
+import queryPackage.QueryClass;
+
+public class TestQueryClass {
+
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+	}
+
+	@Test
+	public void testGetNoOfDeleatedRow() {
+		// there was 1 row matching the condition
+		QueryClass.initConnection();
+		QueryClass.setAutocommit(false);
+		int expected = 1;
+		assertEquals(expected, QueryClass.getNoOfDeleatedRow());
+		QueryClass.rollback();
+		QueryClass.closeConnection();
```

Comment:
ACTION: connection closing should be done in the API and not in the test/implementation